### ISSUE TITLE
Grid Search for parameters

### ISSUE THREAD

**Issue by Prabhakaran-Gokul** [Prabhakaran-Gokul/ROB-498-Robot-Learning#1]
Do grid search on different parameters such as:
1. Number of epoches
2. Network expressivity (Number of layers, Number of neurons per layer)
3. t in odeint function
5. integration algorithm (e.g. dopri5 etc)

Compare performance using:
1. Compare pybullet trajectory with model trajectory
2. Training loss, Validation Loss, Test loss
3. Avg number of steps to goal

**Comment by thirumalaesh-a** [comment]
Done
